bedrock/firefox/templates/firefox/new/basic/thanks.html

```diff
@@ -41,9 +41,7 @@
 
   <section class="mzp-c-call-out mzp-t-product-firefox">
     <div class="mzp-l-content">
-      <h1 class="mzp-c-call-out-title show-windows show-mac show-else">{{ ftl('firefox-new-firefox-is-more-than-a-browser') }}</h1>
-
-      <h1 class="mzp-c-call-out-title show-linux-arm">{{ ftl('firefox-new-download-interrupted') }}</h1>
+      <h1 class="mzp-c-call-out-title show-windows show-mac show-else show-linux">{{ ftl('firefox-new-firefox-is-more-than-a-browser') }}</h1>
```

Comment:
Not a blocker, but might it be worth adding a new string here to match the `thanks.html` template? i.e. "Almost there! Select a download option below"